### DISCUSSION TITLE
Kuscia 上运行 SCQL 联合分析任务时报错

### DISCUSSION THREAD

**Discussion by moon-177** [secretflow/secretflow#1478] [Q&A]
请问在 Kuscia 上运行 SCQL 联合分析任务时，返回以下报错，是什么原因导致的?
其中Kuscia为0.11.0b0，SCQL 为0.9.0b1。同步查询前的操作均按照文档完成（https://www.secretflow.org.cn/zh-CN/docs/kuscia/v0.11.0b0/tutorial/run_scql_on_kuscia_cn#kuscia-scql）。

bash-5.2# curl -X POST http://127.0.0.1:80/intra/query \
--header "host: scql-broker-intra.alice.svc" \
--header "kuscia-source: alice" \
-H "Content-Type: application/json" \
-d '{
    "project_id": "demo",
    "query":"SELECT ta.credit_rank, COUNT(*) as cnt, AVG(ta.income) as avg_income, AVG(tb.order_amount) as avg_amount FROM ta INNER JOIN tb ON ta.ID = tb.ID WHERE ta.age >= 20 AND ta.age <= 30 AND tb.is_active=1 GROUP BY ta.credit_rank;"
}'

{"status":{"code":320,"message":"RunExecutionPlan run jobs(f049da41-704f-11ef-a139-0aab3169f188) failed, catch std::exception=[Enforce fail at engine/datasource/csvdb_adaptor.cc:194] !duck_result->HasError(). send query to DuckDB failed, msg=IO Error: No files found that match the pattern \"/home/kuscia/var/storage/data/scql-alice.csv\"\nStacktrace:\n#0 scql::engine::op::RunSQL::Execute()+0x560c6855c9c8\n#1 scql::engine::Executor::RunExecNode()+0x560c682587e6\n#2 scql::engine::EngineServiceImpl::RunPlanCore()+0x560c68242639\n#3 std::_Function_handler<>::_M_invoke()+0x560c682434fb\n#4 std::__future_base::_State_baseV2::_M_do_set()+0x560c6823c133\n#5 (unknown)+0x7fc896f6fee8\n","details":[]},"result":null}

**Comment by zimu-yuxi** [comment]
在此issue中已解决https://github.com/secretflow/secretflow/issues/1480
